### PR TITLE
test: remove precise length for results

### PR DESCRIPTION
**Pull request by pkey**
Here we don't really care about the length of results and having a set
number for amount of issues makes this test prone to flakiness.  We are more
interested whether the bundle was extended properly.

### CODE REVIEW

**Comment by neonnoon** [comment]
To be honest, I'm not sure I understand what those tests are doing. Are we testing the content that the API returns (i.e., we're testing for the contract), or are we testing local functionality?

**Review by michelkaporin — APPROVED** [review]
Since this is blocking the release currently and I had to change the tests expectations back and forth several times in my last PRs -> LGTM.

**Review by neonnoon — APPROVED** [review]
Approving so we're not blocking, but I think we need to improve our testing principles here.

**Comment by pkey** [comment]
> Why is the test flaky, does it run against codeapi rather than a mock endpoint?

Yes, it does run against `code_api`. To best of my understanding, this is testing whether the files were extended once the endpoint was called... But it means we involve a third party service, which is prone to flakiness.

**Comment by pkey** [comment]
> To be honest, I'm not sure I understand what those tests are doing. Are we testing the content that the API returns (i.e., we're testing for the contract), or are we testing local functionality?

@neonnoon It uploads the files, then, using a thing called extender ([here](https://github.com/snyk/code-client/blob/master/tests/constants/sample.ts#L55)), manipulates the content of files in memory and then tests whether newly uploaded files are being handled correctly by the API.

So, I'd say it does both - tests the contract and tests local functionality. Probably should focus on the latter in this case.

**Comment by pkey** [comment]
That being said, I don't think any of these tests really care about the amount of issues found - I assume we trust what, given the files provided, `suggest` will return.

**Comment by neonnoon** [comment]
[stepsize] Can we separate api contract testing from local unit tests?

File: tests/analysis.spec.ts
Comment (neonnoon) [inline tests/analysis.spec.ts:165]: [stepsize] Can we separate api contract testing from local unit tests?

Comment (pkey) [reply tests/analysis.spec.ts:165]: Next step: bring stepsize back to our lives :D

Comment (neonnoon) [reply tests/analysis.spec.ts:165]: Yes! 😬  One step at a time.

**Pull request merged**
